db_access.py

```diff
@@ -24,16 +30,25 @@ def get_all_users():
     return conn.execute('SELECT id, handle, admin FROM user').fetchall()
 
 
+def user_for(uid):
+    conn = get_db()
+    for a in conn.execute('SELECT handle FROM user WHERE id=:id', {'id':uid}):
+        return a[0]
+    return ''
```

Comment:
I don't think this is needed for the current card, please let me know if I'm missing something important.